pkg/logutil/rate.go

```diff
@@ -0,0 +1,44 @@
+// Copyright 2020 PingCAP, Inc. Licensed under Apache-2.0.
+
+package logutil
+
+import (
+	"fmt"
+	"sync/atomic"
+	"time"
+
+	"github.com/pingcap/log"
+	"go.uber.org/zap"
+)
+
+// TrivialRater is a trivial rate tracer.
+// It doesn't record any time sequence, and always
+// return the average speed over all the time.
+// TODO: replace it with Prometheus.
```

Comment:
what about `ReadCounter` in `pkg/lightning/metric/metric.go`

Reply:
done